pdp/attribute.go

```diff
@@ -403,6 +463,12 @@ func (v AttributeValue) Serialize() (string, error) {
 	case TypeString:
 		return v.v.(string), nil
 
+	case TypeInteger:
+		return strconv.FormatInt(v.v.(int64), 10), nil
+
+	case TypeFloat:
+		return strconv.FormatFloat(v.v.(float64), byte('G'), 16, 64), nil
```

Comment:
If no specific objections the -1 precision is better as it makes `FormatFloat` so many digits that `ParseFloat` returns exactly value `FormatFloat` gets. And you don't need to cast 'G' to byte. The changes make the line to look exactly as [above](https://github.com/infobloxopen/themis/pull/112/files#diff-f4fc264e7d4c0af26700a6f5fc900b41R278).

Reply:
Yes - my bad, again. I thought I changed them all to `-1`.
Will fix.

Reply:
Done